### ISSUE TITLE
Maximum Call stack size exceeded when using CollectionFS url() - 1.3.1 syntax

### ISSUE THREAD

**Issue by vacarsu** [Urigo/angular-meteor#987]
``` javascript
this.helpers({
  image() { return Images.findOne({ _id: imageId }) }
})
```
It still displays the image but going to the route takes a bit longer than it should while it logs maximum call stack size exceeded several times. It also seems to log it every digest cycle.

**Comment by MilosStanic** [comment]
If you've just switched from v1.2 to v1.3.1, check in your code if you still have old style searching for image records with underscore. If you do, it will cause Max Call stack size exceeded error. 
From v1.3 all searches, writes, etc, should be executed directly upon collection.

**Comment by dotansimha** [comment]
@MilosStanic is right, Angular-Meteor 1.2 CollectionFS wrapper breaks the 1.3. 
Also, please try to return the `.url()` value of the image you try to fetch.

**Comment by vacarsu** [comment]
I have updated the entire app to the new syntax, but I am still getting this error.

**Comment by MilosStanic** [comment]
@vacarsu as a personal insight: I'm not using FS.File methods from templates anymore. One of the main reasons was that I figured out that .url() method is not ready immediately upon upload and I wasn't able to fetch images' urls to write them to my other collections. So when I upload, I ping the server to ask if file isStored() and then when it is, I fetch the url from server, and write it in my collection/model. That way I can directly ask the server for images via url, and without calling .url() all the time.

**Comment by Urigo** [comment]
@vacarsu is it an Angular issue or an CollectionFS issue?
Can you share a repo with the problem?

**Comment by Urigo** [comment]
Fixed in `1.3.2`
